conneg-by-ap/index.html

```diff
@@ -281,6 +281,32 @@ <h3>Archival Resource Key (ark)</h3>
 		  </p>
 	  </section>
     <div class="issue" data-number="383"></div>
+    <section id="related-altviews">
+      <h3>Linked Data APIs</h3>
+      <p>
+        Several Linked Data API tools provide access to different representations of resources, identified by URI, via
+        use of Query String Arguments (QSAs). Two examples are the
+        <a href="http://epimorphics.github.io/elda/current/index.html">ELDA tool by Epimorphics</a>, an
+        implementation of the <a href="https://github.com/UKGovLD/linked-data-api/blob/wiki/Specification.md">Linked
+        Data API</a> specification created by the government of the United Kingdom, and the <a href="">pyLDAPI</a> Python
+        Linked Data API implementation by <a href="https://www.csiro.au/">CSIRO</a>. Both ff these tools allow the
+        listing of <em>views</em> and <em>formats</em> available for a resource by dereferencing the resource URI + the
+        QSA <code>_view=alternates</code>. The <em>view</em> named <em>alternates</em> is a special view available for
```

Comment:
Personally, I'm not a big fan of using '+' in running text. I'd propose "by dereferencing the URL created by adding the <abbr title='Query String Argument'</abbr> <code>_view=alternates</code> to the resource URI."